### PR TITLE
Adds brain damage lines about cats

### PR DESCRIPTION
**Pull request by optimumtact**
Is now a brain damage line, along with a few other common pr titles

### CODE REVIEW

**Comment by vcordie** [comment]
EVERY FUCKING TIME I FALL FOR THIS SHIT

**Comment by MollyKristoph** [comment]
SCHROEDINGER WUZ RITE GAS THE KATZ

**Comment by Bawhoppen** [comment]
add humans to the list, I dare you

File: strings/brain_damage_lines.json [hunk 1]
```diff
@@ -71,6 +72,17 @@
         "stun gloves"
     ],
 
+    "removals": [
+        "kats",
+        "liziiggers",
+        "plasmemes",
+        "lizards",
+        "podspeople",
+        "slimescum",
```
Comment (ExcessiveUseOfCobblestone) [inline strings/brain_damage_lines.json]: the gutter-mind test

File: strings/brain_damage_lines.json [hunk 2]
```diff
@@ -71,6 +72,17 @@
...
+        "humons",
+        "bawhoppen"
```
Comment (ExcessiveUseOfCobblestone) [inline strings/brain_damage_lines.json]: who?

**Comment by Armhulen** [comment]
GOT EM

**Pull request closed without merging**